eclipse-theia/theia
# nls: do not localize `open vsx registry`

#### What it does


The commit updates the localization of the string `Open VSX Registry` which was previously translated by Theia.
Given that this is the official name of the service, localization for it's name should not happen.

![Screen Shot 2022-08-03 at 4 27 35 PM](https://user-images.githubusercontent.com/40359487/182705549-331201c5-8218-44d8-8945-45c1e93a9c35.png)

In other areas we correctly localize the registry:

https://github.com/eclipse-theia/theia/blob/5aeef6c0c683b4e91713ab736957e6655b486adc/packages/vsx-registry/src/browser/vsx-extensions-search-bar.tsx#L44

Related: https://github.com/eclipse-theia/theia/discussions/11495.

#### How to test


1. start the application
2. open the extensions-view and search
3. confirm that `Open VSX Registry` is not localized
4. change languages using `Configure Display Language`
5. confirm that `Open VSX Registry` is not localized for different languages

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
